cmd/kpod/load.go

```diff
@@ -39,11 +44,8 @@ func loadCmd(c *cli.Context) error {
 
 	args := c.Args()
 	var image string
-	if len(args) == 1 {
-		image = args[0]
-	}
-	if len(args) > 1 {
-		return errors.New("too many arguments. Requires exactly 1")
+	if len(args) > 2 {
```

Comment:
Why are we allowing 2 args now? Signature policy is handled as a flag, and I don't see any other code changes?

Reply:
I thought the signature-policy was being handled as an argument, but I'll recheck.

Reply:
Hmm, coulda sworn that blew up in testing, but you're right @mheon, I've reverted.